ts/Stock/Indicators/SMA/SMAIndicator.ts

```diff
@@ -297,7 +297,12 @@ class SMAIndicator extends LineSeries {
         const linkedSeriesUnbiner = addEvent(
             Chart,
             'afterLinkSeries',
-            function (): void {
+            function ({ isUpdating }: AnyRecord): void {
```

Comment:
Probably type:
```ts
function (event: { isUpdating: boolean })
```
 would be better. Or perhaps
 
```ts
function (event: Event&{ isUpdating: boolean })
```

Or shall we create a separate interface for it? 🤔 What do you think @bre1470